core/src/test/java/org/apache/iceberg/TestTransaction.java

```diff
@@ -535,7 +535,12 @@ public void testTransactionRetryAndAppendManifests() throws Exception {
     // create a manifest append
     OutputFile manifestLocation =
         Files.localOutput("/tmp/" + UUID.randomUUID().toString() + ".avro");
-    ManifestWriter<DataFile> writer = ManifestFiles.write(table.spec(), manifestLocation);
+    ManifestWriter<DataFile> writer =
+        ManifestFiles.write(
+            table.spec(),
+            manifestLocation,
+            table.properties().get(TableProperties.AVRO_COMPRESSION),
+            table.properties().get(TableProperties.AVRO_COMPRESSION_LEVEL));
```

Comment:
I don't think it is worth updating places like this. It's just needless code churn when we have no reason to think that the codec would be set or matter to the test. Can you remove cases like this from the PR?

Reply:
Reverted and tried reducing the overall churn.